### PR TITLE
add bsize keyword tests v2

### PR DESCRIPTION
**Pull request by jmtaylor90**
This adds signature tests for the various bsize keyword values.

Signed-off-by: jason taylor <jtfas90@gmail.com>

### CODE REVIEW

File: tests/test-bsize-values/test.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+args:
+- -k none
+
+requires:
+   min-version: 7.0.0
```
Comment (catenacyber) [inline tests/test-bsize-values/test.yaml:5]: Does size not work on previous versions ?

**Comment by catenacyber** [comment]
cc @jlucovsky as you work on `bsize` in some PRs, right ?

**Comment by catenacyber** [comment]
cf #238

File: tests/test-bsize-values/test.yaml (hunk 1 above)
Comment (jmtaylor90) [reply tests/test-bsize-values/test.yaml:5]: I will double check, I thought I had 5 as the min version on this.

Comment (jmtaylor90) [reply tests/test-bsize-values/test.yaml:5]: The 6.x checks fail if I lower the version.

Comment (catenacyber) [reply tests/test-bsize-values/test.yaml:5]: Right, `<=` must be min 7 even if bsize < existed before :-)

Comment (jlucovsky) [reply tests/test-bsize-values/test.yaml:5]: Older versions only supported `>` and `<`

@jmtaylor90 One path is to create a test with just these for 6 and newer and then a test with the other operands for 7 and newer.

Comment (jmtaylor90) [reply tests/test-bsize-values/test.yaml:5]: Sure, I can do that. I will resubmit with the appropriate tests split out.

**Comment by jmtaylor90** [comment]
continued in #922

**Pull request closed without merging**